src/Mvc/Mvc.Core/src/ProducesResponseTypeAttribute.cs

```diff
@@ -43,6 +55,19 @@ public ProducesResponseTypeAttribute(Type type, int statusCode)
         /// </summary>
         public int StatusCode { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the default error type should be used when one
+        /// is expliclty provided as a type in the <see cref="ProducesResponseTypeAttribute"/>.
```

Comment:
Nit: if I'm reading this correctly, the flag means the opposite of what the sentence above implies. That is, `true` means we use the explicitly-provided one, and `false` means we use the global one. Also typo in "explicitly".

```suggestion
        /// Gets or sets whether or not the default error type should be overridden when one
        /// is explicitly provided as a type in the <see cref="ProducesResponseTypeAttribute"/>.
```